### PR TITLE
Animal shelter

### PR DESCRIPTION
**Pull request by M-Ravenmoore**
- [ ] Top-level README “Table of Contents” is updated
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
     - [ ] “Happy Path” - Expected outcome
     - [ ] Expected failure
     - [ ] Edge Case (if applicable/obvious)
 - [ ] README for this challenge is complete
     - [ ] Summary, Description, Approach & Efficiency, Solution
     - [ ] Link to code
     - [ ] Picture of whiteboard

### CODE REVIEW

File: javascript/code-challenges/FIFOAnimalShelter/__TEST__/animalShelter.test.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      shelter.enqueue('dog');
+      expect(shelter.dequeue('snake')).toBe(null);
+    });
+    it('if oposit option in preferance it should return what the next animal to be adopted', () =>{
+      const shelter = new AnimalShelter();
+      shelter.enqueue('cat');
+      expect(shelter.dequeue('dog')).toEqual('the next animal is cat');
+    });
+  });
+});
```
Comment (adrianhuebner) [inline javascript/code-challenges/FIFOAnimalShelter/__TEST__/animalShelter.test.js:49]: Some of your tests aren't currently passing

File: javascript/code-challenges/FIFOAnimalShelter/__TEST__/animalShelter.test.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    it('not work with invalid animals', () =>{
+      const shelter = new AnimalShelter();
+      shelter.enqueue('cat');
+      shelter.enqueue('dog');
+      expect(shelter.dequeue('snake')).toBe(null);
+    });
+    it('if oposit option in preferance it should return what the next animal to be adopted', () =>{
+      const shelter = new AnimalShelter();
+      shelter.enqueue('cat');
+      expect(shelter.dequeue('dog')).toEqual('the next animal is cat');
```
Comment (adrianhuebner) [inline javascript/code-challenges/FIFOAnimalShelter/__TEST__/animalShelter.test.js:46]: Like this for example is saying that there is no message coming back that says 'the next animal is cat'
